sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -659,6 +659,14 @@ public partial class RetryOptions
         public System.TimeSpan NetworkTimeout { get { throw null; } set { } }
     }
     public delegate System.Threading.Tasks.Task SyncAsyncEventHandler<T>(T e) where T : Azure.SyncAsyncEventArgs;
+    public partial class TelemetryDetails
+    {
+        public TelemetryDetails(System.Reflection.Assembly assembly, string? applicationId = null) { }
+        public string? ApplicationId { get { throw null; } }
+        public System.Reflection.Assembly Assembly { get { throw null; } }
+        public void ApplyToMessage(Azure.Core.HttpMessage message) { }
```

Comment:
I still prefer just `Apply` here, as I think it reads better in the usage scenario

```csharp
telemetryDetails.Apply(message);
```

vs.

```csharp
telemetryDetails.ApplyToMessage(message);
```

which sounds redundant to me.  I think in .NET we believe users will use Intellisense to understand what the types of input parameters are.  Also, leaving off the `ToMessage` suffix means if we ever wanted to apply telemetry details to something else, we could do it with overloads and keep like methods in a method group.